### PR TITLE
Replace DOM query with ref in Suggester

### PR DESCRIPTION
**Pull request by luisbettencourt**
## 🌟 What is the purpose of this PR?

By @nathggns's suggestion we should avoid using querying the DOM where we can, so we should replace the `querySelector` in the Suggester introduced in #896.

## 🔍 What does this change?

Replaced querySelector with the rootDocumentRef, passed through `createEditorView`.

### CODE REVIEW

File: packages/hash/frontend/src/blocks/page/createSuggester/createSuggester.tsx [hunk 1]
```diff
@@ -122,6 +122,7 @@ export const createSuggester = (
   renderPortal: RenderPortal,
   getManager: () => ProsemirrorManager,
   accountId: string,
+  documentRootRef: RefObject<HTMLDivElement>,
```
[the remark this answers is not in the document]
Comment (luisbettencourt) [reply packages/hash/frontend/src/blocks/page/createSuggester/createSuggester.tsx:125]: Not sure I know what you mean by first argument

[the remark this answers is not in the document]
Comment (nathggns) [reply packages/hash/frontend/src/blocks/page/createSuggester/createSuggester.tsx:125]: My bad, comment on the wrong function.

File: packages/hash/frontend/src/blocks/page/createEditorView.ts [hunk 2]
```diff
@@ -28,14 +29,15 @@ export const createEditorView = (
   accountId: string,
   pageEntityId: string,
   blocks: BlocksMap,
+  documentRootRef: RefObject<HTMLDivElement>,
```
Comment (nathggns) [inline packages/hash/frontend/src/blocks/page/createEditorView.ts:32]: This contains the same node passed in the `renderNode` argument. We should de-duplicate.

Comment (luisbettencourt) [reply packages/hash/frontend/src/blocks/page/createEditorView.ts]: My bad! Didn't realise we were already passing the element there. Done in 248f0e73932a1b7b040fabf10865e4a6521914d1

**Review by nathggns — APPROVED** [review]

**Review by nathggns — APPROVED** [review]

**Pull request merged**